app/models/institution.rb

```diff
@@ -252,7 +252,7 @@ def self.autocomplete(search_term, limit = 6)
     )
   }
 
-  scope :filter, lambda { |field, value|
+  scope :filter_result, lambda { |field, value|
```

Comment:
changed name since another defined scope with the name `filter`, same issue as InstitutionPrograms

Reply:
Nice ambiguity catch 👍